pkg/endpointmanager/conntrack.go

```diff
@@ -122,9 +112,10 @@ func EnableConntrackGC(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint)
 
 	go func() {
 		for {
+			var maxDeleteRatio float64
```

Comment:
Shouldn't this be outside the `for { ...`?

Reply:
The delete ratio is calculated for each run. It's initialized there to make sure it is 0.0 for the per endpoint CT tables. That said, there was a check missing to skip the adjustment if the delete ratio is 0.0. I've added that.